include/task_states.h

```diff
@@ -34,7 +34,7 @@ typedef struct CronTask
 	int64 jobId;
 	int64 runId;
 	CronTaskState state;
-	uint pendingRunCount;
+	unsigned int pendingRunCount;
```

Comment:
I think this is meant to be `uint32` (a type defined by postgres), but fine with this change.